include/rnp/rnp.h

```diff
@@ -2269,6 +2269,22 @@ rnp_result_t rnp_identifier_iterator_next(rnp_identifier_iterator_t it,
  */
 rnp_result_t rnp_identifier_iterator_destroy(rnp_identifier_iterator_t it);
 
+/** Read from input and write to output
+ *
+ *  @param input stream to read data from
+ *  @param output stream to write data to
+ *  @return RNP_SUCCESS on success, or any other value on error
+ */
+rnp_result_t rnp_pipe(rnp_input_t input, rnp_output_t output);
+
+/** Set line length for armored output
+ *
+ *  @param output stream to configure
+ *  @param llen line length in characters
+ *  @return RNP_SUCCESS on success, or any other value on error
+ */
+rnp_result_t rnp_output_armor_set_line_length(rnp_output_t output, int llen);
```

Comment:
Why llen is int, not size_t? Is it supposed to be negative?
Also it would be good to document highest/lowest value (76 and something like 16 or so).